components/ArtPiecePreview.js

```diff
@@ -0,0 +1,11 @@
+import Image from "next/image";
+
+export default function ArtPiecePreview({ image, title, artist }) {
+  return (
+    <div>
+      <h2>{title}</h2>
+      <Image src={image} alt="art-piece" width={240} height={130} />
+      <h3>{artist}</h3>
+    </div>
```

Comment:
Generally we want to avoid divs, since they don't convey any meaning. 
```suggestion
    <article>
      <h2>{title}</h2>
      <Image src={image} alt="art-piece" width={240} height={130} />
      <h3>{artist}</h3>
    </article>
```